### DISCUSSION TITLE
Obstacles in migration from uvicorn cli (package import option, env-file option, additional parameters)

### DISCUSSION THREAD

**Discussion by C-nit** [fastapi/fastapi-cli#127] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
fastapi dev --env-file env/.dev.env --header server:fastapi-server-42 main.py

Usage: fastapi dev [OPTIONS] [PATH]
Try 'fastapi dev --help' for help.
╭─ Error ─────────────────────────────────────╮
│ No such option: --env-file                  │
╰─────────────────────────────────────────────╯
```

### Description

I have an app that works fine with uvicorn cli and
* that uses a lot of environment variables, that are passed via an env file `--env-file`
* starts an app that is not imported in a local file like main.py, but an installed package
* I also pass other parameters that could but are not forwarded to uvicorn

```console
uvicorn --env-file env/dev.env --header server:fastapi-server-42 package_installed_elsewhere.main:app
```

Now maybe it's just not for me and I'll stick to the good old uvicorn cli, but I wanted to give this feedback. I wanted to use the fastapi cli since it's the recommended approach, but was not warned that I might not be able to run the app as before.

### Operating System

Other

### Operating System Details

irrelevant

### Python Version

irrelevant

**Comment by jmehnle** [comment]
I have the same problem. Because I'm using Trio instead of `asyncio`, I'm forced to use hypercorn, and so I can't currently use the `fastapi` CLI, which is a real shame because it provides so much niceties beyond just invoking uvicorn. If only the `fastapi` CLI allowed one to configure the ASGI server to use.
